.github/workflows/win-wheels.yaml

```
@@ -105,7 +105,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["windows-latest"]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

setup-python needs the `-dev` suffix for pre-release versions of Python (there is a discussion about this going on now).
```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11-dev"]
```